src/_common/notification/notification-text.service.ts

```diff
@@ -1,7 +1,7 @@
 import { Community } from '../community/community.model';
 import {
 	CommunityUserNotification,
-	NotificationType,
```

Comment:
Ensure the formatting is in line with the rest of the repository here: We use tailing commas for imports.

Reply:
It's probably some formatting extension not set up on your editor, either editorconfig, prettier or eslint.